DigitalLearningSolutions.Web/ViewModels/LearningPortal/Current/SetLearningLogItemCompletionDateViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+{
+    public class SetLearningLogItemCompletionDateViewModel : SetLearningLogItemCompletionDateFormData
+    {
+        public SetLearningLogItemCompletionDateViewModel(
+            int learningLogItemId,
+            string resourceName
+        )
+        {
+            LearningLogItemId = learningLogItemId;
+            ResourceName = resourceName;
+        }
+
+        public SetLearningLogItemCompletionDateViewModel(
+            SetLearningLogItemCompletionDateFormData formData,
+            int learningLogItemId
+        ) : base(formData)
+        {
+            LearningLogItemId = learningLogItemId;
+        }
+
+        public int LearningLogItemId { get; set; }
+
+        public string ResourceName { get; set; }

```

Comment:
Aren't we going to lose the name on model error since it is on the incorrect model? It's being posted by a hidden in the form, but is on the ViewModel not FormData

Reply:
Oh you're right, didn't notice that. Moved it to the FormData :)